l/libiio/stone.yaml

```diff
@@ -0,0 +1,28 @@
+#
+# SPDX-FileCopyrightText: © 2020-2024 Serpent OS Developers
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+name: libiio
+version: v0.26
+release: 1
+homepage: https://github.com/analogdevicesinc/libiio
+upstreams:
+  - https://github.com/analogdevicesinc/libiio/archive/refs/tags/v0.26.tar.gz: fb445fb860ef1248759f45d4273a4eff360534480ec87af64c6b8db3b99be7e5
+summary: A cross platform library for interfacing with local and remote Linux IIO devices
+description: |
+  libiio is used to interface to the Linux Industrial Input/Output (IIO) Subsystem.
+builddeps:
+  - binary(bison)
+  - binary(flex)
+  - libaio-devel
+  - pkgconfig(avahi-glib)
+  - pkgconfig(libusb-1.0)
+  - pkgconfig(libxml-2.0)
+license: LGPL-2.1 GPL-2.0
```

Comment:
`license` needs to be moved up above `summary`, and needs to be a valid SPDX license identifier. There are several examples in the repo.

Reply:
It is not constant where the license line is placed:
ent: below description
inxi: below description
kdb: below description
itstool: below description
python-pyyaml: below description
zlib: below summary